### ISSUE TITLE
[Feature Request] Add the ability to destruct the editor

### ISSUE THREAD

**Issue by hsalem7** [Ionaru/easy-markdown-editor#341]
If there is a place where we are conditionally showing the editor, then it is better to have a destruct method.

```javascript
var simplemde = new SimpleMDE({ element: document.getElementById("MyID") });
```

And when there is a need to hide/remove the editor:
```javascript
simplemde.destruct();
```

It should remove all the elements that were added because of the construct method.

**Comment by hsalem7** [comment]
I found that this function already exists, and it is called `toTextArea()`!. What do you think if we rename it? Or add a clearer description?

**Comment by Ionaru** [comment]
Renaming the `toTextArea()` function is not possible in any `2.x.x` version because it would break backwards compatibility. Maybe an alias is acceptable?

**Comment by hsalem7** [comment]
> Renaming the `toTextArea()` function is not possible in any `2.x.x` version because it would break backwards compatibility. Maybe an alias is acceptable?

Yeah that would be great, or to add a comment block to describe how toTextarea can be used, thankss

**Comment by Ionaru** [comment]
It is mentioned in the readme. https://github.com/Ionaru/easy-markdown-editor#removing-easymde-from-text-area

**Comment by vanillajonathan** [comment]
Close the issue if it is resolved.
